README.md

```diff
@@ -183,15 +183,16 @@ More ready-to-use examples can be found in the [Example](https://mosecorg.github
 
 ## Deployment
 
-- This may require some shared memory, remember to set the `--shm-size` flag if you are using docker.
-- This service doesn't require Gunicorn or NGINX, but you can certainly use the ingress controller. BTW, it should be the PID 1 process in the container since it controls multiple processes.
+- If you're looking for a GPU base image with `mosec` installed, you can check the official image [`mosecorg/mosec`](https://hub.docker.com/r/mosecorg/mosec). For the complex use case, check out [envd](https://github.com/tensorchord/envd).
+- The deep learning service may require some shared memory, remember to set the `--shm-size` flag if you are using docker.
+- This service doesn't need Gunicorn or NGINX, but you can certainly use the ingress controller. BTW, it should be the PID 1 process in the container since it controls multiple processes.
```

Comment:
```suggestion
- This service doesn't need Gunicorn or NGINX, but you can certainly use them as ingress controllers inside the container. If so, make sure your ingress controller is the PID 1 process since it controls multiple processes.
```

Reply:
Should not use them as the PID 1 since they don't guarantee other processes can get the signal.

Ingress controllers are usually used in the Kubernetes cluster, which is reasonable. If you need to use nginx before mosec but in one place, it should be different containers in a pod (for cluster deployment) or they should be started with a multiprocess-manager (local container mode).

Reply:
Then maybe putting is as another bullet point is better?

Reply:
Agree. Will create a new PR for this.